backend/setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import find_packages, setup
+
+install_requires = [
```

Comment:
Should we put explicit version numbers in the setup.py file and lock Pipfile to the same ones?